app/routes/circulars/circulars.lib.ts

```diff
@@ -132,15 +132,15 @@ export function formatIsValid(format: string): format is CircularFormat {
   return (circularFormats as any as string[]).includes(format)
 }
 
-/** For updated dates, check that the date is valid */
-export function dateIsValid(date?: string, time?: string) {
-  return !Number.isNaN(Date.parse(`${date}T${time}:00.000Z`))
-}
-
 export function submitterIsValid(submitter?: string) {
   return Boolean(submitter)
 }
 
+export function dateTimeIsValid(date: string) {
+  if (!date) return true
```

Comment:
I think that this may be incorrect because the date is _invalid_ if it is an empty string. Also this is redundant because `Date.parse('')` returns `NaN`.

```suggestion
```

Reply:
An undefined date IS valid for new circulars. In the new circular route there will not be a createdOn date. I moved the coercing it as valid for create back to the component.

Reply:
I like that solution. Thank you!